securedrop/manage.py

```diff
@@ -18,8 +19,11 @@
 from db import db_session, init_db, Journalist
 from management.run import run
 
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
+log = logging.getLogger(__name__)
 
-def reset():  # pragma: no cover
+
+def reset(args):  # pragma: no cover
```

Comment:
I created this PR to reduce the scope of the other PRs and gradually merge them. I did not do that originally (this PR was part of https://github.com/freedomofpress/securedrop/pull/1850) but after a few weeks with no review activity it seemed like a good idea to split it in this way. I'm happy to re-integrate it back to https://github.com/freedomofpress/securedrop/pull/1850 if you think this is the best course of action.